TestParticleMakie/src/interactive.jl

```diff
@@ -5,36 +5,38 @@
 
 A plot recipe for plotting orbit or other figures related to six phase space coordinates and time.
 
-# Arguments:
-- `sol::AbstractODESolution`: a solution returned by solver.
-- `vars`: a argument used to choose the variables to be plotted. Default value is [(1, 2, 3)]
+# Arguments
+- `sol::AbstractODESolution`: the solution returned by the ODE solver.
+
+# Keywords
+- `vars`: the argument used to choose variables to be plotted. Default value is [(1, 2, 3)].
 - `tspan::Tuple`: the span of time to be plotted. For example, tspan = (0, 1).
-- `to_3d::Bool`: whether to force the points to be plotted in 3d. If the switch is on, the order of coordinates will be rearranged for correctly plotted as 3d points.
+- `to_3d::Bool`: whether to force the points to be plotted in 3d. If `true`, the order of coordinates will be rearranged for plotting as 3d points.
 - `interactive::Bool`: whether to show the figure in interactive mode.
 
 ## vars
-`vars` can be a Integer, Function, Tuple and Array. The basic form for `vars` is Tuple. For example, if you want to plot the variable `x` as a function of time or the orbit of a particle, you can use it like this:
+`vars` can be an Integer, Function, Tuple and Array. The basic form for `vars` is Tuple. For example, if you want to plot the variable `x` as a function of time or the orbit of a particle, you can use it like this:
 ```julia
 orbit(sol, vars=(0, 1))
 orbit(sol, vars=(1, 2, 3))
 ```
-If the length of the tuple is 3, it will be converted to 3d 
-If you want to plot a function of time, position or velocity, you can define a function firstly. The arguments of the function must be a 7-dimensional array, the elements of which is corrspending to the phase space coordinates and time at a certain time respectively, and the return value must be a Number. For example,
+If the length of the tuple is 3, it will be converted to 3d.
+If you want to plot a function of time, position or velocity, you can first define a function. The arguments of the function must be a 7-dimensional array, the elements of which correspond to the time and phase space coordinates, and the return value must be a Number. For example,

```

Comment:
Maybe the order between time and phase space coordinates should be reversed. Because the argument of the function corresponds ['x', 'y', 'z', "vx", "vy", "vz", 't']. My reason for putting 't' at the end is that in most cases such functions do not involve time, when the first six variables correspond to phase space coordinates more naturally, so putting the time at the end prevents the user from paying extra attention to the presence of this variable.